### DISCUSSION TITLE
Can we Test the Webpage Using Quali ?

### DISCUSSION THREAD

**Discussion by Vignesh-Tester** [QualiTorque/qualitorque.github.io#261] [Q&A]
Can we test the Webpage using Quali Torque or Quali Cloudshell?
For example, after entering the inputs, I have to validate the login page.

Is this can be done? If yes, how? Please help me with this.

**Comment by shirabendor-quali** [comment]
Hi @Vignesh-Tester ,
I'm not sure that I understand your issue 100%...
Quali Torque and Cloudshell are about providing you with the infrastructure which your app needs, with self service and governance. Validating the login page sounds to me as part of your application? or I might misunderstand. 
Please can you clarify your question, Thanks

**Reply by Vignesh-Tester** [reply]
Hi @shirabendor-quali , 
My Query is I need to test the Webpage UI Functionality. Will Quali assist me with that?

In Simple, Is the Quali tool will be useful for testing purposes?

**Reply by shirabendor-quali** [reply]
@Vignesh-Tester yes it's useful for testing purposes. If you wish to see a demo please reply and we can set it up.

**Reply by Vignesh-Tester** [reply]
Yes, I have a trial license ( Quali tool) but In the documentation, I can't find help regarding how quali will help the tester to test the application and develop a script and Frameworks. (Like selenium or accelQ) 
my mail Id is: vptvignesh13@gmail.com
